data-processing/bot.py

```
@@ -0,0 +1,17 @@
+# bot.py  (only on_message handler changed)
+import json, discord, os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)